src/runtime_src/core/pcie/linux/pcidev.cpp

```diff
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: Apache-2.0
-// Copyright (C) 2016-2020 Xilinx, Inc
-// Copyright (C) 2022 Advanced Micro Devices, Inc. All rights reserved.
+// Copyright (C) 2016-2020, 2023 Xilinx, Inc
```

Comment:
I don't usually update Xilinx Inc on copyright these days.  Are we supposed to?

Reply:
Oh, no. It's automatically done by my vim script. I need to fix it. We should update the AMD one.

Reply:
That is a great idea to have such a script anyway!